nilearn/plotting/find_cuts.py

```diff
@@ -210,7 +210,15 @@ def find_cut_slices(img, direction='z', n_cuts=12, spacing='auto'):
     data = _smooth_array(data, affine, fwhm='fast')
 
     if spacing == 'auto':
-        spacing = max(int(.5 / n_cuts * data.shape[axis]), 1)
+        if n_cuts > 0.:
+            spacing = max(int(.5 / n_cuts * data.shape[axis]), 1)
+        else:
+            message = ("If the input 'cut_coords' is an integer, it only "
+                       "represents the number of slices to display. It should "
+                       "be given with value more than 0. or If you want to "
```

Comment:
Two things come to mind:
a) Is it intuitive to the user that you write "0" as float?
b) Language: probably new sentence before second If here.